src/Actions/Payments/GetPaymentQrCodeAction.php

```
@@ -0,0 +1,43 @@
+     *
+     * @param  string  $paymentId  The ID of the payment to retrieve the QR code for.
+     * @return array The QR code details including encoded image, payload, expiration date, and description.
+     *
+     * @throws \InvalidArgumentException if the provided payment ID is empty.
+     * @throws ApiException
+     * @throws AuthenticationException
+     * @throws NotFoundException
+     * @throws RateLimitException
+     * @throws ValidationException
+     */
+    public function handle(string $paymentId): array
```

PHPStan (2.1.31)

Method AsaasPhpSdk\Actions\Payments\GetPaymentQrCodeAction::handle() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)